.gitignore

```diff
@@ -14,6 +14,14 @@ npm-debug.log
 /lib/*
 !/lib/.npmignore
 
+/authProviders/*
+
+src/**/*.js
+src/**/*.js.map
+src/**/*.d.ts
+
+samples/**/*/secrets.ts
+samples/**/*/secrets.js
```

Comment:
Can you give a few sample paths that this covers? Most likely you won't need the last stars.

Reply:
Corrected the glob to samples/**/secrets.x. 
The secret.sample files should be renamed to -> example - secrets.ts and the sample code imports {credentials} from '/secrets.file'. The secrets.sample files are dummy files with placeholders which should be renamed and replaced by someone checking out the sample. The .gitignore avoids a git push of a file with the actual credentials values.